setup.py

```diff
@@ -16,6 +16,7 @@
     "redis-py-cluster": ["redis-py-cluster>=2.1.2,<3.0.0"],
     "refcycle": ["objgraph>=3.0,<4.0"],
     "requests": ["advocate>=1.0.0,<2.0"],
+    "s3fetcher": ["boto3>=1.18"],
```

Comment:
ngl this is a bit arbitrary. We're using pretty basic s3 features and I scanned back through the changelog to see if there were notable events. We tested with 1.20